pkg/lib/authn/otp/sender.go

```diff
@@ -105,6 +106,9 @@ func (s *MessageSender) setupTemplateContext(msg *PreparedMessage, opts SendOpti
 			linkURL = s.Endpoints.ResetPasswordEndpointURL()
 			query := linkURL.Query()
 			query.Set("code", opts.OTP)
+			if opts.IsAdminAPIResetPassword {
+				query.Set("x_from_admin_api", "true")
```

Comment:
Make this a constant, and use it in webapp

Reply:
Updated, thanks!